packages/components/src/components/FormInput/FormInput.tsx

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Label } from "../Label";
+import { Input } from "../Input";
+import { HelpText } from "../HelpText";
+import type { InputProps } from "../Input";
+import { Box } from "../../primitives/Box";
+
+export interface FormInputProps extends Omit<InputProps, "id"> {
+  /** The `id` of the input. */
+  id: string;
+  /** The text to be used for Label. */
+  label: string;
+  /** The text to be used for HelpText. */
+  helpText?: string;
```

Comment:
These should both be react nodes similar to the typing for children in the components.